genref/config.yaml

```diff
@@ -56,90 +61,108 @@ apis:
       - k8s.io/controller-manager/config/v1alpha1
 
   - name: kube-proxy-config
+    title: kube-proxy configuration (v1alpha1)
     package: k8s.io/kube-proxy
     path: config/v1alpha1
 
   - name: metrics
+    title: ""
     package: k8s.io/metrics
     path: pkg/apis/metrics/v1alpha1
 
   - name: metrics
+    title: ""
```

Comment:
Why is this title empty?

Reply:
Not sure what to use for this now. Can propose a temp title.